frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -32,6 +32,9 @@
 
 .ceph-add-capacity_sc-dropdown {
```

Comment:
correct this too

Reply:
what needs to corrected here?

Reply:
`ceph-add-capacity__sc-dropdown` , as per the convention.

Reply:
ack